### PR TITLE
added proguard metadata

### PR DESCRIPTION
**Pull request by marandaneto**
## :loudspeaker: Type of change

- [ ] Bugfix
- [X] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description
adding proguard metadata to the event

## :bulb: Motivation and Context
we need the proguard file id to deobfuscate stacktraces

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] All tests passing

## :crystal_ball: Next steps
how do I use sentry-cli?

### CODE REVIEW

File: sentry.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+defaults.project=android
+defaults.org=man
+auth.token=
```
Comment (marandaneto) [inline sentry.properties]: @bruno-garcia we might need a take token where the server will not refuse it

File: build.gradle.kts [hunk 2]
```diff
@@ -22,6 +22,7 @@ buildscript {
         classpath(Config.BuildPlugins.androidGradle)
         classpath(kotlin(Config.BuildPlugins.kotlinGradlePlugin, version = Config.kotlinVersion))
         classpath(Config.QualityPlugins.errorpronePlugin)
+        classpath("io.sentry:sentry-android-gradle-plugin:1.7.28")
```
Comment (bruno-garcia) [inline build.gradle.kts]: Can this be only for the sample instead?

File: sentry.properties (hunk 1 above)
Comment (bruno-garcia) [reply sentry.properties]: We wouldn't want to check-in an auth token though. I'm not sure we could limit it only for uploading proguard files so.
We can set this via environment variable, and `sentry-cli` will pick it up

Comment (marandaneto) [reply sentry.properties]: ok I kept it commented for now and let's discuss it later on

File: build.gradle.kts (hunk 2 above)
Comment (marandaneto) [reply build.gradle.kts]: @bruno-garcia this is only for the sample, classpath means available, who wants to add the plugin.

I kept it commented for now and let's discuss it later on

**Pull request merged**